configure.ac

```
@@ -5992,6 +5992,11 @@ AS_VAR_IF([with_readline], [no], [
       AC_DEFINE([HAVE_RL_APPEND_HISTORY], [1], [Define if readline supports append_history])
     ])
 
+    # in readline as well as newer editline (April 2023)
+    AC_CHECK_TYPE([rl_compdisp_func_t], [
+      AC_DEFINE([HAVE_RL_COMPDISP_FUNC_T], [1], [Define if readline supports rl_compdisp_func_t])
+    ], [], [readline_includes])
```

Style nit; please follow the [GNU Autoconf style guide](https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.71/html_node/Coding-Style.html).

```suggestion
    AC_CHECK_TYPE([rl_compdisp_func_t],
                  [AC_DEFINE([HAVE_RL_COMPDISP_FUNC_T], [1],
                             [Define if readline supports rl_compdisp_func_t])],
                  [],
                  [readline_includes])
```

Even though configure.ac is coloured with a variety of styles, I prefer to stick with a clean style for all new code.

---

Sorry, fixed now!

I indeed just copy-pasted the exact style of the readline checks before it.